src/web3_utils.py

```diff
@@ -79,23 +80,24 @@ def get_strategies_and_vaults(
 
     for version in vaults_by_version.keys():
         for vault_address in vaults_by_version[version].keys():
-            strategy, vault = get_strategy_from_vault(
-                node, chain, vault_address, version=version
-            )
-            vaults.append(
-                Contract(
-                    name=vaults_by_version[version][vault_address]["name"],
-                    contract=vault,
-                    address=vault_address,
+            if vault_address not in ETH_REGISTRY_SETTINGS.externally_managed_vaults:
```

Comment:
Same here, would like to see a comment about why we don't earn externally_managed_vaults